lib/applicants/validation.ts

```diff
@@ -27,7 +27,14 @@ export const requestPermitHolderInformationSchema = object({
   phone: string()
     .required('Please enter a valid phone number')
     .max(12, 'Please enter a valid phone number in the format 000-000-0000'),
-  receiveEmailUpdates: boolean().required(),
+  receiveEmailUpdates: boolean()
+    .nullable()
+    .default(null)
+    .when('type', {
+      is: 'NEW',
+      then: boolean().required('Please enter your date of birth'),
```

Comment:
Incorrect error message

Reply:
See comment above (no error message was indicated in the rebased version, which makes sense considering it's a checkbox)